### DISCUSSION TITLE
Migrating Restructured Text (.rst) to a Jekyll based site?

### DISCUSSION THREAD

**Discussion by epugh** [jekyll/jekyll#9651] [Q&A] [answered]
Curious if anyone has tips/tools/suggestions on migration RestructuredText files to something that Jekyll supports?  I found a very out of date plugin for Jekyll, but nothing current.  Are there RST --> MD translators that folks have used?

**Comment by fauno** [comment, marked as answer]
You could use [Pandoc](https://pandoc.org/index.html) for initial conversion to Markdown. If you want to keep the RST files you'll need a Jekyll generator plugin (I don't remember if it supports it natively).

**Comment by epugh** [comment]
The Jekyll one that I found is https://github.com/xdissent/jekyll-rst and looks abandonded....   However, I trialed Pandoc...   and Wow...   that is pretty darn amazing!

Thanks!
